### PR TITLE
Added GQuery drag and drop plugin documentation

### PR DESCRIPTION
**Pull request by Elrhino**

### CODE REVIEW

**Comment by Elrhino** [comment]
Should we keep .jar files, and live examples from google code?

File: site/src/main/markdown/gquery/config.xml [hunk 1]
```diff
@@ -7,36 +7,22 @@
         style="gquery">
     <toc>
         <entries>
-            <entry name="index" title="Overview" displayName="Overview"
```
Comment (olafleur) [inline site/src/main/markdown/gquery/config.xml]: Watch out! You reverted @jasonlemay 's changes, I think.

**Comment by Elrhino** [comment]
@olafleur, @jasonlemay - I fixed the revert I made to your changes on `config.xml`.

**Comment by olafleur** [comment]
LGTM ! ;-)

**Pull request merged**